src/stores/songs-store.js

```diff
@@ -0,0 +1,70 @@
+import { observable, action } from 'mobx'
+import axios from 'axios'
+
+class SongsStore {
+
+  @observable albumTracks = []
+  @observable clickedAlbum = ''
+  @observable clickedSong = ''
+  @observable songToDisplay = {}
+
+  @action
+  updateAlbumTracks(value) {
+    this.albumTracks = value
+  }
+
+  @action
+  updateClickedAlbum(value) {
```

Comment:
Nie do końca rozumiem czemu zarządzanie albumem jest w SongsStore. Piosenka należy do albumu a nie albumy do piosenek

Reply:
zmienilem nazwe na albumStore, ma wiecej sensu

Reply:
Ogólnie wydaje mi się, że to powinny być dwa różne store'y. Na ten moment jest ok bo store jest mały, ale w przyszłości może się szybko rozrastać bo ma za dużo odpowiedzialności.

Reply:
Hmm, no jak juz mowilem nie planuje, zeby sie rozrastalo, ale jak bede dodawac cos wiecje, to fakt, moge zrobic album i pojedynczy song store, ale poki co jest dosc maly tak sadze

Reply:
Ok ;)